composer.json

```diff
@@ -14,7 +14,11 @@
     "require": {
         "php": "^7.3",
         "symfony/framework-bundle": "^4.4|^5.0",
-        "halloverden/symfony-oidc-contracts": "^1.0"
+        "halloverden/symfony-oidc-contracts": "^1.0",
+        "halloverden/symfony-security": "^1.4",
```

Comment:
might as well wait for halloverden/symfony-security 2.0 and use that before releasing this.